message_structure.md

```diff
@@ -0,0 +1,31 @@
+## Message Structure
+
+DIDComm Messages are based on JWM (JSON Web Messages). A message has a basic structure that specifies the message type, id, and other attributes common to all messages. A message also includes attributes specific to the message type. Prior to being sent to a recipient, the JWM is encrypted into a JWE according to the JWM spec.
+
+The following example shows common elements of a DIDComm Message. Further details and advanced usage are covered elsewhere in this spec.
+
+```json
+{
+    "@id": "1234567890",
+    "@type": "<message-type-uri>",
+    "from": "did:example:alice",
+    "to": ["did:example:bob"],
+    "created_time": "1516269022",
+    "expires_time": "1516269022",
+    "body": {
+    	"messagespecificattribute": "and it's value"
+	}
+}
+```
+
+
+
+The attributes of a DIDComm Message are as follows:
+
+- **id** - REQUIRED. Message ID. The `id` attribute value MUST be unique to the sender.
+- **type** - REQUIRED. Message Type. The `type` attribute value MUST be a valid [Message Type URI](protocols.md#message-type-uri) , that when resolved gives human readable information about the message. The attributes value also informs the content of the message, for example the presence of other attributes and how they should be processed.
```

Comment:
```suggestion
- **@id** - REQUIRED. Message ID. The `@id` attribute value MUST be unique to the sender.
- **@type** - REQUIRED. Message Type. The `@type` attribute value MUST be a valid [Message Type URI](protocols.md#message-type-uri) , that when resolved gives human readable information about the message. The attributes value also informs the content of the message, for example the presence of other attributes and how they should be processed.
```

Reply:
Good catch! I believe this should be updated the other way - by removing the `@` prefix in the examples. This does have implications about JSON-LD compatibility that we have not discussed. I'll put this specific topic on the agenda for Monday's meeting and we can update appropriately to the decisions there.